attestation/src/attestation.rs

```diff
@@ -20,105 +24,260 @@ pub struct DstackAttestation {
     quote: Quote,
     collateral: Collateral,
     tcb_info: TcbInfo,
+    expected_measurements: ExpectedMeasurements,
```

Comment:
Very late post-merge question: Why are the expected measurements part of the `DstackAttestation`? It seems to me like we're now allowing a node to submit arbitrary expected measurements to the smart contract, when in reality we want the contract to have these hard-coded.

Reply:
https://github.com/near/mpc/issues/940